nhs/data/handling.py

```diff
@@ -103,4 +106,68 @@ def read_spreadsheets(
             lambda x: x[0], placeholder_matches(files, file_dir_pattern, ["key"])
         )
 
-    return {key: val for key, val in zip(keys, map(reader, files))}
+    return {key: reader(file, sheet_id) for key, file in zip(keys, files)} if extension == "xlsx" else {
+        key: reader(file) for key, file in zip(keys, files)
+    }
+
+
+@log_entry_exit(level="INFO")
+def standarise_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
+                     identification: str, abbreviation_column_name: str,
+                     long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+    """
+    Process the column names of a polar Lazy frame dictionary to make them more readable.
+        1. Expand abbreviated names.
+        2. Converting to lower case.
+    If any parameters are None, this function returns prev_csv_pl_df without change.
+    The filename in both dictionaries are checked or filtered to be passed.
+
+    Parameters:
+    --------
+    param1 : dict[str, pl.LazyFrame]
+        df_dict: A dictionary containing census data as value and filename as key or None.
+    param2 : [pl.LazyFrame | None]
+        census_metadata: Census metadata lazy frame with one standard sheet or None.
+        Blank files won't change prev_csv_pl_df.
+    param3 : str
+        identification: Identify different files in df_dict. Could not be None
+    param4 : str
+    abbreviation_column_name: column name of abbreviated names. Could not be None
+    param5 : str
+    long_column_name: column name of expanded names. Could not be None
```

Comment:
Replace with actual name of the parameters instead of `param<int>`, so like

```python
"""
df_dict : dict[str, pl.LazyFrame]
        A dictionary containing census data as value and filename as key or None.
"""
```

Reply:
add: all column names in the dataframes in `df_dict`